backend/bot/index.js

```diff
@@ -0,0 +1,60 @@
+var SlackBot = require('slackbots');
+
+var bot = new SlackBot({
+  token: process.env.FOOSBOT_TOKEN
+});
+
+let channel;
+
+console.log('lez go');
+
+bot.on('start', async () => {
+  console.log('Foosbot started');
+  channel = await bot.getGroup(process.env.FOOS_CHANNEL_NAME);
+  if (!channel) {
+    throw `Channel '${process.env.FOOS_CHANNEL_NAME}' not found!`
+  }
+});
+
+const postResultToSlack = (match) => {
+  const { team1, team2, team1Won, ratingChange } = match;
+  let winningTeam, losingTeam;
+  if (team1Won) {
+    [winningTeam, losingTeam] = [team1, team2];
+  } else {
+    [winningTeam, losingTeam] = [team2, team1];
+  }
+  const winningPlayers = winningTeam.map(player => `${player.name} (${player.rating})`)
+  const losingPlayers = losingTeam.map(player => `${player.name} (${player.rating})`)
+
+  let isComedyDuo = true;
+  if (winningPlayers.length != 2) {
+    isComedyDuo = false;
+  }
+  winningTeam.forEach(player => {
+    if (player.name !== 'Pepa' && player.name !== 'Tonda') {
+      isComedyDuo = false;
+    }
+  })
+
+  let prefix = '';
+  if (isComedyDuo) {
+    prefix = ':tondab: :pepadab: ';
```

Comment:
Move the trailing space to the messageText? (so that it doesn't need to be repeated)

Reply:
Then there would be empty space at the start of the `messageText` if the prefix ends up being empty.

Reply:
Makes sense... then add it to 'HOLY SHIT!', where it's currently missing please 😄

Reply:
Oh, it's already there... nevermind!